### ISSUE TITLE
sugar cane technology never worked with AUGER WAGON itself

### ISSUE THREAD

**Issue by MarckAmorim** [Courseplay/courseplay#6660]
what is up? whenever I use any trailer it gets jammed with the cane cutting machinery and its AUGER WAGON don't even recognize

**Comment by 60erjonny** [comment]
Hallo

Doesn't help your problem, but I use the Annaburger HTS 34.16 Mod and have had good experiences ...
mfg
jonny

**Comment by MarckAmorim** [comment]
I try user this modEm 11 de jan de 2021 08:48, 60erjonny <notifications@github.com> escreveu:
Hallo
Doesn't help your problem, but I use the Annaburger HTS 34.16 Mod and have had good experiences ...
mfg
jonny

**Comment by Tensuko** [comment]
#4021

**Comment by mricher-git** [comment]
I used the Fliegl Dolly mod with the Fligle ASS 298 mod, the trailer is low enough the harvester doesn't hit it, and the dolly has a joint which provides enough flex the harvester's tracks stay on the ground. There are many problems with sugarcane, the game treats it like a forage harvester. The big issue for trailers is that, the fillAimTarget moves further back along the trailer the more full it gets. As you go around a corner, the pipe is trying to aim at the back of the trailer, so the pipe goes outside of the trailer boundaries. I created a mod for the it always aims at the front end to fix this problem. It not does a complete field with 14m turn radius and never stops or gets stuck. The regular forage harvester can aim it's pipe so it doesn't have this problem.

If you look at this pic, without my mod, the pipe would aim along the red line and miss the trailer.
I'll attach it here in case you want to give it a try
[FS19_VG_Sugarcane_Autoaim_Fix.zip](https://github.com/Courseplay/courseplay/files/5806029/FS19_VG_Sugarcane_Autoaim_Fix.zip)

![Sugarcane](https://user-images.githubusercontent.com/65866023/104399249-41f01400-558b-11eb-9435-3ff3d2602213.jpg)
